src/test/store/profile-view.test.js

```diff
@@ -755,6 +755,14 @@ describe('snapshots of selectors/profile-view', function() {
     dispatch(ProfileView.changeSelectedCallNode(0, [A, B]));
     dispatch(ProfileView.changeSelectedMarker(0, 1));
     dispatch(ProfileView.addRangeFilter(3, 7)); // Reminder: upper bound "7" is exclusive.
+    dispatch(
+      ProfileView.updateProfileSelection({
+        hasSelection: true,
+        isModifying: false,
+        selectionStart: 4,
+        selectionEnd: 6,
+      })
+    );
```

Comment:
This is a quite small test change, please tell me if you'd like some more test in the test file for the component `CallTreeSidebar` too.

Reply:
This is fine.